frontend/models/components/company.ts

```diff
@@ -15,7 +15,10 @@ export function companyFromStrapi(
    const name = fragment?.attributes?.name;
    const logo = imageFromStrapi(fragment?.attributes?.logo);
 
-   if (name == null || logo == null) return null;
+   if (name == null || logo == null) {
+      console.warn('company name and/or logo missing!');
+      return null;
+   }
```

Comment:
This is intentional. It's to have a warning for when name or logo are not set in Strapi for a company. When there's no image the company won't be rendered in the press quote section.
I've wasted a bit of time due to this, so hopefully this would help someone else understand why a press quote is not rendered. I know it's not the best, and I'm open for suggestions, but in the context of this PR I though this was sufficient